### DISCUSSION TITLE
Varying Output Noise Example

### DISCUSSION THREAD

**Discussion by ericpena** [GPflow/GPflow#2034] [Q&A]
Hello, I've been learning how to use GPFlow recently and it's fantastic! The [GPR with Varying Output Noise Demo 3](https://gpflow.github.io/GPflow/2.6.4/notebooks/advanced/varying_noise.html#Demo-3:-Empirical-noise-variance) example is particularly helpful for my use case.

The explanation for this Demo 3 is very clear:
> "The shaded gray area is the confidence interval of f. We use vertical lack lines to indicate the confidence interval of the Y mean. [...] You may notice that the data points fall outside our confidence interval. Again, this is because the confidence interval is for the mean of Y, and not the Y points themselves."

My silly question is: If I did indeed want to retrieve the confidence interval of the Y points themselves, could this involve doing a Bayesian fit to estimate the variance for each distribution (i.e., the distributions at the each x-value) independently?
Although, I believe this analysis would give me a credible interval instead of a confidence interval but nevertheless, I'm just trying to understand this "Y points themselves" piece a bit more.

Thanks for any help you can provide!

**Comment by frazane** [comment]
Hi @ericpena, any news regarding this question? I'm dealing with a similar situation.
